README.md

```diff
@@ -58,21 +58,21 @@ echo "deb https://dl.bintray.com/clevercloud/deb unstable beta" | tee -a /etc/ap
 
 ##### Warning
 
-Bintray is ending on May 1st 2021, we will migrate hosting as soon as possible. Please refer to the dedicated [issue](https://github.com/CleverCloud/clever-tools/issues/454) to track the migration.
+We recently moved from Bintray to a self hosted Nexus repository available at [https://nexus.clever-cloud.com](https://nexus.clever-cloud.com). You have to update the URL of your repository with the new one. If you experience any issue please let us know on the following [issue https://github.com/CleverCloud/clever-tools/issues/454](https://github.com/CleverCloud/clever-tools/issues/454).
 
 ---
 
 If you are using a GNU/Linux distribution that uses `.rpm` packages like CentOS or Fedora, you can run:
 
 ```sh
-curl https://bintray.com/clevercloud/rpm/rpm > /etc/yum.repos.d/bintray-clevercloud-rpm.repo
-echo "exclude=*beta*" >> /etc/yum.repos.d/bintray-clevercloud-rpm.repo
+curl https://nexus.clever-cloud.com/repository/rpm/ > /etc/yum.repos.d/nexus-clevercloud-rpm.repo
```

Comment:
Do not work. Just save the nexus html page into `nexus-clevercloud-rpm.repo` instead of a configuration

Reply:
This works but needs to be improved for the gpg side:

Create `/etc/yum.repos.d/clever-tools.repo` with following content:

```
[clever-tools]
name=Clever Tools
baseurl=https://nexus.clever-cloud.com/repository/rpm/
enabled=1
gpgcheck=0
gpgkey=
```